pkg/server/grpc/tracee.go

```diff
@@ -274,21 +283,16 @@ func getThreat(e trace.Event) *pb.Threat {
 		}
 	}
 
-	if _, ok := e.Metadata.Properties["external_id"]; ok {
-		if val, ok := e.Metadata.Properties["external_id"].(string); ok {
-			mitreTechniqueExternal = val
-		}
-	}
```

Comment:
So actually this is the field we should assign to mitreTechniqueId (T1234) and not the "id" property (we will probably want to change that in the future for all signatures to be "id")